### DISCUSSION TITLE
Option to only use convoke the spirits with celestial alignment

### DISCUSSION THREAD

**Discussion by Olanid57** [Hekili/hekili#966] [Ideas]
Hello !

In a lot of fights (depending of fight duration), top parsers always delay covoke to line up with CA.

As explain on dreamgrove : "Yes you should always delay Convoke UNLESS you can cast a Convoke without losing a CA+ Convoke (both at the same time) cast. This will for example happen in 5 minute fights where you would cast Convoke+CA on pull, Convoke alone after 2 minutes and then Convoke+CA at 4 minutes."

I try to do this by toggling cooldowns OFF/ON in fight but an option is better.

Can you add it to hekili or is there a way to do it in other way please ?

**Comment by Hekili** [comment]
You can do this by:
1. Toggling CDs.
2. Toggling Covenant abilities.
3. Writing a custom priority.

**Reply by LazlovGit** [reply]
Same as a Shadow Priest. A option to only show spells/CDs when other spells/CDs are available. Hekili would be the perfect Addon with this option. For Example: Only show Mindbender + Power Infusion when Void Eruption is up (or will be in the next 5 Sec.)

**Reply by Hekili** [reply]
The same answer applies.  If that logic is built into the sim priority, it's already integrated into the addon.  If not, you'd need to write your own custom priority and do your own testing for it.
